citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -140,6 +180,31 @@ $(function() {
 	    }, 1000);
 	});
 	
+	$(".author-item").on("click", function(){
```

Comment:
let's add an 'edit' icon to edit an author instead of clicking on the whole thing.